doc/sphinx-guides/source/installation/prerequisites.rst

```diff
@@ -322,27 +322,35 @@ components and libraries. Please consult the instructions in the
 Installing R
 ============
 
-Can be installed with :fixedwidthplain:`yum`::
+For RHEL/CentOS, the EPEL distribution is strongly recommended::
 
-       yum install R-core R-core-devel
+If :fixedwidthplain:`yum` isn't configured to use EPEL repositories ( https://fedoraproject.org/wiki/EPEL ):
 
-EPEL distribution is strongly recommended. The version of R currently available from epel6 and epel7 is 3.5; it has been tested and is known to work on RedHat and CentOS versions 6 and 7.
+RHEL/CentOS 8 users can install the epel-release RPM::
 
-If :fixedwidthplain:`yum` isn't configured to use EPEL repositories ( https://fedoraproject.org/wiki/EPEL ):
+       yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
 
-RHEL/CentOS users can install the RPM :fixedwidthplain:`epel-release`. For RHEL/CentOS 7::
+RHEL/CentOS 7 users can install the epel-release RPM::
 
        yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
 
-RHEL/CentOS users can install the RPM :fixedwidthplain:`epel-release`. For RHEL/CentOS 6::
+RHEL 8 users will need to enable the CodeReady-Builder repository::
+
+       subscription-manager repos --enable codeready-builder-for-rhel-8-x86_64-rpms
 
-       yum install https://dl.fedoraproject.org/pub/epel/epel-release-latest-6.noarch.rpm
+CentOS 8 users will need to enable the PowerTools repository::
 
-RHEL users will want to log in to their organization's respective RHN interface, find the particular machine in question and:
+       dnf config-manager --enable PowerTools
```

Comment:
I'm new to PowerTools as well. I'm starting to worry that there are a lot of steps here. Is it possible to simplify it, at least in how it's described at the top as "You're going to enable EPEL and then install a package called R-core-devel." Or the extra things, PowerTools and CodeReady-Builder really necessary?

Reply:
yes, the R portion became a bit of a choose-your-own adventure, but to my knowledge these are the necessary steps. I forget the additional dependencies provided by each repo. I can write a friendlier introductory paragraph if you'd like (but first let me comment out these steps in dataverse-ansible to verify that they're still needed).

Reply:
Yes, we still need these repos: `Depsolve Error occured: \n Problem: conflicting requests\n  - nothing provides openblas-devel needed by R-core-devel-4.0.3-1.el8.x86_64\n  - nothing provides texinfo-tex needed by R-core-devel-4.0.3-1.el8.x86_64`